lms/templates/instructor/instructor_dashboard_2/cohort-form.underscore

```diff
@@ -0,0 +1,99 @@
+<% var isNewCohort = cohort.id == null; %>
+<div class="cohort-management-settings">
+    <form action="" method="post" name="" id="cohort-management-settings-form" class="cohort-management-settings-form">
+        <% if (isNewCohort) { %>
+            <h3 class="form-title"><%- gettext('Add a New Cohort Group') %></h3>
+        <% } %>
+
+        <div class="form-fields">
+            <%
+            // Don't allow renaming of existing cohorts yet as it doesn't interact well with
+            // the course's advanced setting for auto cohorting.
+            if (isNewCohort) {
+            %>
+                <div class="form-field">
+                    <div class="cohort-management-settings-form-name field field-text">
+                        <label for="cohort-name" class="label">
+                            <%- gettext('Cohort Group Name') %> *
+                            <span class="sr"><%- gettext('(Required Field)')%></span>
+                        </label>
+                        <input type="text" name="cohort-name" value="<%- cohort ? cohort.get('name') : '' %>" class="input cohort-name"
+                               id="cohort-name"
+                               placeholder="<%- gettext("Enter the name of the cohort group") %>" required="required" />
+                    </div>
+                </div>
+            <% } %>
+
+            <%
+            var foundSelected = false;
+            var selectedContentGroupId = cohort.get('group_id');
+            var hasSelectedContentGroup = selectedContentGroupId != null;
+            var hasContentGroups = contentGroups.length > 0;
+            %>
+            <div class="form-field">
+                <div class="cohort-management-details-association-course field field-radio">
+                    <label class="label">
```

Comment:
Good catch, not sure why that was there. It's a heading now. However, it looks like the class `.label` is pre-existing and used in `.member-list-widget`. Not sure how pervasive this is, but I've added an additional class to the styles for this particular case.